packages/browser-data-collector/src/report.ts

```diff
@@ -51,14 +51,26 @@ export class ReportImpl implements Report {
 		}
 	}
 
-	private async start() {
-		await Promise.all( this.startCollectors.map( ( collector ) => collector( this ) ) );
+	private async runCollectors( collectors: Collector[] ) {
+		return Promise.all(
+			collectors.map( ( collector ) => {
+				try {
+					collector( this );
```

Comment:
Is this missing a `return`? I think this `map` will just result in an array of undefined otherwise.

Reply:
Good catch, fixed!